Dockerfile

```
@@ -0,0 +1,4 @@
+FROM openjdk:11-jre-slim
+ARG JAR_FILE=build/libs/*.jar
+COPY ${JAR_FILE} app.jar
```

```suggestion
ADD ./build/libs/*.jar app.jar
```
굳이 arg로 받을 필요는 없을거 같고 요렇게 치환?

---

#7a5d4ee 에 반영했습니다.